### DISCUSSION TITLE
How to get new language definitions into my project?

### DISCUSSION THREAD

**Discussion by behoppe** [PrismJS/prism#3527] [Q&A]
Hello, I am happily using Prismjs in my 11ty site https://www.opencilk.org. A colleague @ailiop has added two new languages to Prismjs that have recently been merged (#3522) and I am struggling to incorporate those updates into my project. I am new to Node, so I apologize if this question doesn't belong here.

I use https://github.com/pborenstein/eleventy-md-syntax-highlight to interface with Prismjs, and it throws an error, "cilkc is not a valid Prism.js language for eleventy-md-syntax-highlight," when I try to use the new `cilkc` language implemented in #3522 (by downloading a new CSS from https://prismjs.com/download.html#themes=prism&languages=markup+css+clike+c+cpp+cilkc+cilkcpp&plugins=line-numbers, which seems give me a good CSS btw).

In my Node project, I see in `/node_modules/prismjs/components` that indeed `cilkc` is not there. I have tried `npm update @pborenstein/eleventy-md-syntax-highlight` and `npm update prismjs` to no effect. I have also created a Node project from scratch and installed Prismjs as the only package in the project, and still `cilkc` is not present in the `/node_modules/prismjs/components`  folder.

How do I access the new language definitions implemented in #3522? Thank you very much.

**Comment by RunDevelopment** [comment]
Hello!

We haven't published #3522 yet. We plan to make a new release this weekend, so you'll be able to use it soon. 

Unfortunately, there is nothing you can do until then, so you'll have to wait. Sorry.

**Reply by behoppe** [reply]
OK, thank you very much. I can wait :-)
